src/util/context.js

```diff
@@ -0,0 +1,3 @@
+import { createContext } from "react";
+
+export const CartContext = createContext("");
```

Comment:
Since this is being used as a "component", I wonder if this should be in the components folder instead.

Reply:
it's more like a wrapper than a component.  Very similar to Redux.